pynest/nest/__init__.py

```diff
@@ -93,11 +93,17 @@ def __init__(self, name):
         _rel_import_star(self, ".lib.hl_api_types")
 
         # Lazy loaded modules. They are descriptors, so add them to the type object
-        type(self).spatial = _lazy_module_property("spatial")
         type(self).raster_plot = _lazy_module_property("raster_plot")
+        type(self).spatial = _lazy_module_property("spatial")
         type(self).visualization = _lazy_module_property("visualization")
         type(self).voltage_trace = _lazy_module_property("voltage_trace")
 
+        try:
+            _rel_import_star(self, ".server.hl_api_server")
+            type(self).server = _lazy_module_property("server")
+        except ImportError:
```

Comment:
Since it is lazy loaded there is no importing going on in this try block, and no `ImportError` will ever be raised. The try/except can be removed. The `ImportError` will be thrown on the first time `nest.server` is accessed in user code. Which seems like good behavior, right?